{
  "comments": [
    {
      "key": {
        "uuid": "7a1969be_405ed2ea",
        "filename": "automated/utils/test-runner.py",
        "patchSetId": 1
      },
      "lineNbr": 462,
      "author": {
        "id": 1000015
      },
      "writtenOn": "2017-03-15T14:55:17Z",
      "side": 1,
      "message": "should we add \u0027test-run\u0027: \u0027pass\u0027 if the condition is not met?",
      "revId": "456492e652e750d1d3f55982cf4ef7bd39f90383",
      "serverId": "f33910f19b7abb192b83adbd000000bf",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "7a1969be_a0918654",
        "filename": "automated/utils/test-runner.py",
        "patchSetId": 1
      },
      "lineNbr": 462,
      "author": {
        "id": 1000096
      },
      "writtenOn": "2017-03-16T06:58:19Z",
      "side": 1,
      "message": "I think the purpose of showing \u0027test-run fail\u0027 in result file is to remind result reviewer that sth was wrong here.\n\nWhen test finished correctly, IMHO, we don\u0027t need this item.",
      "parentUuid": "7a1969be_405ed2ea",
      "revId": "456492e652e750d1d3f55982cf4ef7bd39f90383",
      "serverId": "f33910f19b7abb192b83adbd000000bf",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "7a1969be_db1637f1",
        "filename": "automated/utils/test-runner.py",
        "patchSetId": 1
      },
      "lineNbr": 462,
      "author": {
        "id": 1000015
      },
      "writtenOn": "2017-03-16T13:12:48Z",
      "side": 1,
      "message": "We would be on par with \u0027old LAVA\u0027 where test-run pass/fail was recorded always.",
      "parentUuid": "7a1969be_a0918654",
      "revId": "456492e652e750d1d3f55982cf4ef7bd39f90383",
      "serverId": "f33910f19b7abb192b83adbd000000bf",
      "unresolved": false
    }
  ]
}